contracts/src/management/ManagementContract.sol

```diff
@@ -113,15 +118,16 @@ contract ManagementContract is Ownable, Initializable {
         require(isAggAttested);
 
         if (verifyAttester) {
+            
             // the data must be signed with by the correct private key
             // signature = f(PubKey, PrivateKey, message)
             // address = f(signature, message)
             // valid if attesterID = address
-            bytes32 calculatedHashSigned = ECDSA.toEthSignedMessageHash(abi.encodePacked(attesterID, requesterID, hostAddress, responseSecret));
+            bytes32 calculatedHashSigned = abi.encodePacked(attesterID, requesterID, hostAddress, responseSecret).toEthSignedMessageHash();
```

Comment:
this was the issue?

Reply:
nope, its in the go code where the signature is generated; We need to append to correct recovery id.